commands-yml/commands/element/actions/click.yml

```diff
@@ -34,7 +34,12 @@ example_usage:
       $driver->byAccessibilityId('SomeId')->click();
   csharp:
     |
-      // TODO
+      MobileElement el = driver.FindElement(MobileByAccessibilityID("Some ID");
+      el.click();
+
+      //Example
+      MobileElement el = driver.FindElement(MobileById("Some ID");
```

Comment:
Thanks!
Do you think here needs two examples?

Reply:
@KazuCocoa - In most of the documentation, there are two code snippets (One Syntax and One Example). Feel free to remove one snippet from the code if you think it's unnecessary.

Reply:
Okay. np for this.